packages/core/src/customer/strategies/stripe-upe/stripe-upe-customer-strategy.ts

```diff
@@ -109,8 +111,12 @@ export default class StripeUPECustomerStrategy implements CustomerStrategy {
                 appearance,
             });
 
-            const billingAddress = this._store.getState().billingAddress.getBillingAddress();
-            const { email: billingEmail } = billingAddress || {};
+            const {
+                billingAddress: { getBillingAddress },
+                consignments: { getConsignmentsOrThrow },
+            } = this._store.getState();
+            const { id } = getConsignmentsOrThrow()[0] || '';
```

Comment:
Do we need to give a default empty string value?

Reply:
@animesh1987 done, I also replace the `getConsignmentsOrThrow` function to avoid any future issue, let me know if you like this approach :D